addons/payment/views/payment_portal_templates.xml

```diff
@@ -160,6 +153,7 @@
                                           class="col-md-9 col-form-label"
                                           t-esc="tx.amount"
                                           t-options="{'widget': 'monetary', 'display_currency': tx.currency_id}"/>
+                                    <t t-call="payment.transaction_status"/>
```

Comment:
This should replace the `div` that you remove By placing the `t-call` here, the alert block is placed *after* the payment information, which is not what we want.

Reply:
Understood!